src/SFA.DAS.AANHub.Application/Common/Validators/CreateMemberCommandValidator.cs

```diff
@@ -19,6 +19,25 @@ public CreateMemberCommandValidator()
                 .Matches(Constants.RegularExpressions.EmailRegex);
             RuleFor(x => x.Joined)
                 .NotEmpty();
+            RuleFor(x => x.Regions).Custom((regions, context) =>
```

Comment:
This should be an async validator checking against the database records.

Reply:
Implemented